tests/__init__.py

```diff
@@ -5,13 +5,8 @@
 import os
 import aiounittest
 
-# Before import from the searx package, we need to set up the (debug)
-# environment.  The import of the searx package initialize the searx.settings
-# and this in turn takes the defaults from the environment!
 
 os.environ.pop('SEARXNG_SETTINGS_PATH', None)
-os.environ['SEARXNG_DEBUG'] = '1'
```

Comment:
Why is this no longer needed? Is that because it's already set in `settings.yml`?

Reply:
> Why is this no longer needed? I

Its an leftover (see chapter "Unit Tests" in https://github.com/searxng/searxng/pull/4183)

>  Is that because it's already set in `settings.yml`?

Nope, the settings for the unit test are loaded from [tests/unit/settings](https://github.com/searxng/searxng/blob/master/tests/unit/__init__.py)